web-app/src/components/BookingModalForm/index.js

```diff
@@ -40,70 +43,104 @@ const BookingModalForm = props => {
     }
   };
 
-  const updateVisability = () => {
-    return Math.floor(
-      moment.duration(formData.end.diff(formData.start)).asDays(),
-    ) === 0
-      ? false
-      : true;
+  const calculateDaysNotice = () => {
```

Comment:
I can see this method being used in other places in the app - maybe abstract it out into a util?

Reply:
method has been abstracted out to utilities/dates.js